File: problemas/iniciante/1113/1113.exs
defmodule Modulo do
  def resolve() do
    entrada = IO.gets("") |> String.trim |> String.split(" ")
    numeros = Enum.map(entrada, &String.to_integer/1)

    x = Enum.at(numeros, 0)
    y = Enum.at(numeros, 1)

    cond do
      x < y -> IO.puts "Crescente"
      x > y -> IO.puts "Decrescente"
      true -> :ok
    end

    if x != y do
      resolve()
    end
  end
end

Modulo.resolve()

File: problemas/iniciante/1113/1113.hs
import Text.Printf

compara :: IO ()
compara = do
  entrada <- getLine
  let [x, y] = map read (words entrada) :: [Double]
  if x == y
    then return ()
    else do
      if x < y
        then printf "Crescente\n"
        else printf "Decrescente\n"
      compara

main :: IO ()
main = compara
